pkg/virtual/framework/forwardingregistry/rest_test.go

```diff
@@ -110,9 +110,8 @@ func newStorage(t *testing.T, clusterClient kcpdynamic.ClusterInterface, apiExpo
 		nil,
 		clusterClient,
 		patchConflictRetryBackoff,
-		func(_ schema.GroupResource, store *forwardingregistry.StoreFuncs) *forwardingregistry.StoreFuncs {
-			return store
-		})
+		forwardingregistry.StorageWrapperFunc(func(_ schema.GroupResource, store *forwardingregistry.StoreFuncs) {
```

Comment:
You have an inherent `nil` check - can we just omit this?

Reply:
If we keep the nil check in NewStorage, yes